pkg/tlsprofile/tlsprofile_test.go

```
@@ -24,11 +24,12 @@ import (
 
 func TestTLSOptionsFromProfile(t *testing.T) {
 	tests := []struct {
-		name               string
-		profile            *configv1.TLSSecurityProfile
-		expectedMinVersion string
-		expectCiphers      bool
-		expectError        bool
+		name                   string
```

golangci-lint (2.11.4)

[error] File is not properly formatted

(gofmt)